### DISCUSSION TITLE
Unable to install mlc-ai-nightly on m2 mac air

### DISCUSSION THREAD

**Discussion by darkTemplar** [mlc-ai/mlc-en#46] [General]
When I run `pip install mlc-ai-nightly -f https://mlc.ai/wheels` I get the following error:
`ERROR: Could not find a version that satisfies the requirement mlc-ai-nightly (from versions: none)
  ERROR: No matching distribution found for mlc-ai-nightly
`
When I look at the https://mlc.ai/wheels link, seems like there is no distribution for Apple silicon, though I see there is a commit in the github.com/ml-ai/utils repo last month to add a script to create a wheel file for Apple Silicon.

Would really appreciate some guidance to help resolve this issue.

**Comment by tqchen** [comment]
please check your python there are apple silicon wheels for python 3.10 and 3.11 based on the link

**Reply by darkTemplar** [reply]
Thanks for your reply. I have python 3.10.10 and pip version 23.1, however still not working. I will try and debug more on my end.

**Reply by tqchen** [reply]
could it relates to os version see the wheel here https://github.com/mlc-ai/utils/releases/download/v0.9.dev0/mlc_ai_nightly-0.12.dev785%2Bgfc09f562f-cp310-cp310-macosx_13_0_arm64.whl

**Reply by darkTemplar** [reply]
yep this was the underlying issue. I had mac os 12.5, once I upgraded to 13.3 (ventura), I managed to install the binary linked above. Thanks so much for your help!

**Reply by tqchen** [reply]
glad it works out
